src/ResourceManager/DataLakeStore/Commands.DataLakeStore.Test/App.config

```diff
@@ -0,0 +1,12 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
```

Comment:
Why is this needed?

Reply:
This is for the binding redirect to 9.0.1 since my sdk depends on 6.0.8